src/main/java/recipeapplication/controllers/RecipeController.java

```diff
@@ -22,7 +18,7 @@ public RecipeController(RecipeService recipes) {
 		this.recipes = recipes;
 	}
 
-	@RequestMapping(value = "/recipe", method = RequestMethod.POST)
+	@RequestMapping(value = "/postRecipe", method = RequestMethod.POST)
```

Comment:
leave the routes as in the design, the routes should be the same for every request, the distinguishement is made by the HTTP request itself

Reply:
Edited the routes for the POST and PUT methods to /recipe

@RequestMapping(value = "/recipe", method = RequestMethod.POST)
@RequestMapping(value = "/recipe", method = RequestMethod.PUT)